frontend/components/meta.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import { Helmet } from 'react-helmet'
+// eslint-disable-next-line
 // @ts-ignore
 import favicon from '../images/favicon.png'
```

Comment:
switching to `ts-expect-error` will satiate eslint, but my editor was going nuts over it (which kinda makes sense given this is a raster file) so elected to just do an inline ignore